drivers/math/Make.defs

```diff
@@ -20,7 +20,7 @@
 
 # Include math acceleration support
 
-ifneq ($(CONFIG_MATH_CORDIC)$(CONFIG_MATH_FFT),)
+ifneq ($(CONFIG_MATH_CORDIC)$(CONFIG_MATH_FFT)$(CONFIG_MATH_MPI),)
 CSRCS += math.c
 endif
 
```

Comment:
Please include also the update in the CMakefile, we need to keep both building systems in sync!